docs/tutorials/04_torch_qgan.ipynb

```diff
@@ -716,13 +533,15 @@
     }
    },
    "source": [
-    "### Training\n",
+    "## 5. Model Training\n",
+    "In the training loop we monitor not only loss functions, but relative entropy as well. The relative entropy describes a distance metric for distributions. Hence, we can use it to benchmark how close/far away the trained distribution is from the target distribution.\n",
+    "\n",
```

Comment:
I am looking at the output and see the plots. Do you think any commentary of what we see in the plots, following them, would help the reader or are plots like this normal for the audience and they will be able to interpret what they see in them.

Reply:
Honestly, I think it is okay. There's a description of the relative entropy and this is what we show on the plot. Loss functions are pretty standard for the GAN type of networks. So, I don't think we need more explanations here.